<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassDisplay</class>
 <widget class="QWidget" name="ClassDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QSpinBox" name="m_pBlueSB">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="m_pRedSB">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KColorValueSelector" name="m_pColorPickerR">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="hue">
      <number>0</number>
     </property>
     <property name="saturation">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KColorValueSelector" name="m_pColorPickerG">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="hue">
      <number>0</number>
     </property>
     <property name="saturation">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="m_pGreenSB">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="KColorValueSelector" name="m_pColorPickerB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>30</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="hue">
      <number>0</number>
     </property>
     <property name="saturation">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Background</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorValueSelector</class>
   <extends>QWidget</extends>
   <header>kcolorvalueselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
